Description: skip racy test
Author: James Clarke
Forwarded: no
Last-Update: 2022-09-12

---

--- a/test/redirect.c
+++ b/test/redirect.c
@@ -188,7 +188,7 @@ static int no_redirect(void)
 
     ONN("redirect non-NULL after non-redir req", ne_redirect_location(sess));
 
-    CALL(process_redir(sess, "/foo", &loc));
+    PRECOND(process_redir(sess, "/foo", &loc) == OK);
 
     return destroy_and_wait(sess);
 }
